modules/op-release-notes-1-19.adoc

```diff
@@ -0,0 +1,396 @@
+// This module is included in the following assemblies:
+// * release_notes/op-release-notes-1-19.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="op-release-notes-1-19_{context}"]
+= Release notes for {pipelines-title} General Availability 1.19
+
+With this update, {pipelines-title} General Availability (GA) 1.19 is available on {OCP} 4.15 and later versions.
+
+[id="new-features-1-19_{context}"]
+== New features
+
+In addition to fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.19:
+
+[id="pipelines-new-features-1-19_{context}"]
+=== Pipelines
+
+* With this update, you can now specify custom `securityContext` settings under the `EventListener` YAML specification. When custom `securityContext` is enabled, user-defined values override the default configuration. If no custom values are provided, the default `securityContext` setting is applied automatically.
++
+.Example configuration for enabling custom `securityContext` in the `EventListener` YAML spec
+[source,yaml]
+----
+spec:
+  serviceAccountName: tekton-triggers-example-sa
+  resources:
+    kubernetesResource:
+      spec:
+        template:
+          spec:
+            securityContext:
+              runAsNonRoot: true
+            containers:
+            - resources:
+                requests:
+                  memory: "64Mi"
+                  cpu: "250m"
+                limits:
+                  memory: "128Mi"
+                  cpu: "500m"
+              securityContext:
+                readOnlyRootFilesystem: true
+----
+
+* With this update, you can configure custom database credentials for {tekton-results} using the `TektonConfig` custom resource (CR). This eliminates the need to rely on default PostgreSQL secrets that use default usernames and passwords.
```

Comment:
Is there an example for how to pull in a secret, or would such an example belong in a piece of documentation separate from the release notes?

Reply:
Hi @aThorp96, I am not aware of an example that can be used for this. If either you or Shiv Verma can help me with a piece of code, I can accommodate it and add to the content.

Reply:
Updated on google doc